spynnaker8/models/connectors/multapse_connector.py

```diff
@@ -22,23 +23,29 @@ class MultapseConnector(_BaseClass):
     """
     __slots__ = []
 
-    def __init__(self, num_synapses, allow_self_connections=True,
-                 with_replacement=True, safe=True, verbose=False):
+    def __init__(self, n, allow_self_connections=True,
+                 with_replacement=True, safe=True, verbose=False,
+                 rng=None):
         super(MultapseConnector, self).__init__(
-            num_synapses=num_synapses,
-            allow_self_connections=allow_self_connections,
-            with_replacement=with_replacement, safe=safe, verbose=verbose)
+            num_synapses=n, allow_self_connections=allow_self_connections,
+            with_replacement=with_replacement, safe=safe, verbose=verbose,
+            rng=rng)
 
```

Comment:
The base class doesn't have rng= in the arguments, so this falls over at the moment - I guess it should just be added to the list in the base class in sPyNNaker and then probably set self._rng in there too?

Reply:
Oops - this was in my sPyNNaker changes, but not here - I will make a matching branch...

Reply:
Added support in SpiNNakerManchester/sPyNNaker#528